action.yml

```diff
@@ -0,0 +1,23 @@
+name: slack-user-bot
```

Comment:
Could you rename this file to more meaningful name ex. `post_slack_message_action.yml`

Reply:
i cant. Its required to be this name